contrib/systems.md/thermostat.md

```diff
@@ -1,32 +1,67 @@
 Thermostat
 ==========
 
-A simple thermostat in maude, which switches between on/off based on some threshold temperatures.
+A simple thermostat in Maude, which switches between on/off based on some threshold temperatures.
 The state components of the thermostate are the current time, the temperature, and the mode (on, off, turning on, turning off).
 
+Parametric Thermostat
+---------------------
+
+### Unital Ordered Rings
+
+The thermostat is parametric in an ordered ring with unit (which will be the data for the temperature).
+
 ```maude
-load ../tools/fvp/numbers.maude
+fth UNITAL-ORDERED-RING is
+
+    sorts RingBool Ring .
```

Comment:
I noticed you use a `RingBool` sort instead of hardcoding to `true`/`false`. Is that for SMT purposes?

Reply:
Well, it is instantiated to the SMT solvers as well. But I guess not all users of `true` and `false` use the same sort, so it may be good to have the user call out the correct sort explicitly.

Reply:
That's totally fine by me. I was just double-checking.